plugins/modules/hetzner_dns_record_set.py

```diff
@@ -129,10 +129,10 @@
     zone: foo.com
     record: foo.com
     type: CAA
-    ttl: 3600
+    ttl: null
     value:
-    - "128 issue letsencrypt.org"
-    - "128 iodef mailto:webmaster@foo.com"
+    - "128 issue \"letsencrypt.org\""
+    - "128 iodef \"mailto:webmaster@foo.com\""
```

Comment:
If you use `\` here, you need to change line 12 of the file to `DOCUMENTATION = r'''`.

It is probably better to write the above as
```suggestion
    - '128 iodef "mailto:webmaster@foo.com"'
```
or
```suggestion
    - >-
      128 iodef "mailto:webmaster@foo.com"
```
though.